### ISSUE TITLE
我训练了，为什么没保存模型

### ISSUE THREAD

**Issue by songmimi** [argman/EAST#351]
Step 001200, model loss 0.0122, total loss 0.0309, 3.68 seconds/step, 3.81 examples/second
Step 001210, model loss 0.0123, total loss 0.0309, 3.63 seconds/step, 3.85 examples/second
Step 001220, model loss 0.0140, total loss 0.0325, 3.38 seconds/step, 4.14 examples/second
Step 001230, model loss 0.0159, total loss 0.0343, 3.24 seconds/step, 4.32 examples/second
Step 001240, model loss 0.0138, total loss 0.0322, 3.83 seconds/step, 3.65 examples/second
Step 001250, model loss 0.0112, total loss 0.0295, 3.26 seconds/step, 4.29 examples/second
Step 001260, model loss 0.0133, total loss 0.0315, 3.96 seconds/step, 3.54 examples/second
Step 001270, model loss 0.0154, total loss 0.0336, 3.96 seconds/step, 3.54 examples/second
Step 001280, model loss 0.0126, total loss 0.0306, 4.27 seconds/step, 3.28 examples/second
Step 001290, model loss 0.0110, total loss 0.0290, 3.50 seconds/step, 4.00 examples/second
Step 001300, model loss 0.0134, total loss 0.0313, 3.77 seconds/step, 3.71 examples/second
Step 001310, model loss 0.0237, total loss 0.0416, 3.98 seconds/step, 3.52 examples/second
Step 001320, model loss 0.0162, total loss 0.0340, 3.85 seconds/step, 3.64 examples/second
Step 001330, model loss 0.0166, total loss 0.0344, 3.28 seconds/step, 4.26 examples/second
，我已经运行到这里了。为甚么我没有保存模型，是我做错了吗
tf.app.flags.DEFINE_integer('input_size', 512, '')
tf.app.flags.DEFINE_integer('batch_size_per_gpu', 14, '')
tf.app.flags.DEFINE_integer('num_readers', 24, '')
tf.app.flags.DEFINE_float('learning_rate', 0.0001, '')
tf.app.flags.DEFINE_integer('max_steps', 100000, '')
tf.app.flags.DEFINE_float('moving_average_decay', 0.997, '')
tf.app.flags.DEFINE_string('gpu_list', '0', '')
tf.app.flags.DEFINE_string('checkpoint_path', '/east_icdar2015_resnet_v1_50_rbox/', '')
tf.app.flags.DEFINE_boolean('restore', True, 'whether to resotre from checkpoint')
tf.app.flags.DEFINE_integer('save_checkpoint_steps', 1000, '')#1000
tf.app.flags.DEFINE_integer('save_summary_steps', 100, '')
tf.app.flags.DEFINE_string('pretrained_model_path', 'temp/resnet_v1_50.ckpt', ''）
这是我的参数

**Comment by skywalk163** [comment]
试试运行到5000以上会不会存储ckpt，我的是到了5387step 才存盘的，不明白为什么这么怪。
代码里看着挺正常的。

**Comment by Primarysf** [comment]
从哪里看各项数据的变化呢
